### PR TITLE
Translate the abstract factory readme to Portuguese

### PR DESCRIPTION
**Pull request by ricardo-ramos-moura**
I'll translate the readme file of the abstract-factory design pattern to Portuguese and place it under localization.pt.abstract-factory.

### CODE REVIEW

**Comment by ricardo-ramos-moura** [comment]
@iluwatar could you please review

File: localization/pt/abstract-factory/README.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
+---
+layout: pattern
+title: Abstract Factory
+folder: abstract-factory
+permalink: /patterns/abstract-factory/
+categories: Creational
+language: pt
+tags:
+ - Gang of Four
+---
```
Comment (iluwatar) [inline localization/pt/abstract-factory/README.md:10]: Please have a look at the updated yaml frontmatter requirements in https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute

Or you can just copy them from the main abstract-factory pattern

**Review by iluwatar — CHANGES_REQUESTED** [review]

File: localization/pt/abstract-factory/README.md (hunk 1 above)
Comment (ricardo-ramos-moura) [reply localization/pt/abstract-factory/README.md]: I updated the yaml frontmatter.

File: localization/pt/abstract-factory/README.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+título: Abstract Factory
+categoria: Criacional
+idioma: pt
+tags:
+- Gang of Four
```
Comment (iluwatar) [inline localization/pt/abstract-factory/README.md:6]: I'm sorry but you are not supposed to translate this. You should define it exactly like in the original pattern:

```
---
title: Abstract Factory
category: Creational
language: en
tags:
 - Gang of Four
---
```

Comment (iluwatar) [reply localization/pt/abstract-factory/README.md:6]: Except for the language of course which should be `pt` like you already have

**Comment by iluwatar** [comment]
Part of #2348

**Review by iluwatar — APPROVED** [review]

**Comment by iluwatar** [comment]
Looks good! Thank you for the contribution 🎉

**Pull request merged**